PSPrintSDK/OLProductTemplateSyncRequest.m

```diff
@@ -56,13 +56,72 @@ - (void)fetchTemplatesWithURL:(NSURL *)url templateAccumulator:(NSMutableArray *
                             id identifier = template[@"template_id"];
                             id costs = template[@"cost"];
                             id imagesPerSheet = template[@"images_per_page"];
+                            id product = template[@"product"];
                             NSNumber *enabledNumber = template[@"enabled"];
                             BOOL enabled = enabledNumber == nil ? YES : [enabledNumber boolValue];
                             
                             if ([name isKindOfClass:[NSString class]]
                                 && [identifier isKindOfClass:[NSString class]]
                                 && [costs isKindOfClass:[NSArray class]]
-                                && (imagesPerSheet == nil || [imagesPerSheet isKindOfClass:[NSNumber class]])) {
+                                && (imagesPerSheet == nil || [imagesPerSheet isKindOfClass:[NSNumber class]])
+                                && (product == nil || [product isKindOfClass:[NSDictionary class]])) {
+                                
+                                NSString *coverPhoto;
+                                NSArray *productShots;
+                                NSString *productClass;
+                                UIColor *labelColor;
+                                CGSize sizeCm = CGSizeMake(0, 0);
+                                CGSize sizeInches = CGSizeMake(0, 0);
+                                NSString *code;
+                                if (product){
+                                    coverPhoto = [product[@"ios_sdk_cover_photo"] isKindOfClass:[NSString class]] ? product[@"ios_sdk_cover_photo"] : nil;
+                                    
+                                    NSString *productShot1 = [product[@"ios_sdk_product_shot_1"] isKindOfClass:[NSString class]] ? product[@"ios_sdk_product_shot_1"] : nil;
```

Comment:
Totally agree, I even put them in an array bellow. But that's what I got from the server.